lib/perl/Genome/Model/CwlPipeline/Command/ProcessActionQueue.pm

```diff
@@ -0,0 +1,74 @@
+package Genome::Model::CwlPipeline::Command::ProcessActionQueue;
+
+use strict;
+use warnings;
+
+use Genome;
+
+class Genome::Model::CwlPipeline::Command::ProcessActionQueue {
+    is => 'Command::V2',
+    has => [
+        max_count => {
+            is => 'Number',
+            doc => 'Process at most this many queued builds',
+            default => 10,
+        },
+        submit_jobs => {
+            is => 'Boolean',
+            default => 0,
+            doc => 'if set, will use `genome analysis-project possible-volumes` to submit jobs for the actions',
+        },
+    ],
+};
+
+sub help_detail {
+    return <<EOHELP
+Takes the actions requested for CWLPipeline builds.
+EOHELP
+}
+
+sub execute {
+    my $self = shift;
+
+    my @to_process = Genome::Model::Build::CwlPipeline->get(
+        action_requested => ['stop', 'abandon'],
+        -limit => $self->max_count,
```

Comment:
what does this leading dash do?

Reply:
It's part of the parameter name.  It's a UR-internal parameter so it uses a leading dash to avoid conflicting with any potential property names on the object itself.